engine/src/main/java/org/terasology/logic/nameTags/NameTagClientSystem.java

```diff
@@ -85,7 +85,7 @@ private void createOrUpdateNameTagFor(EntityRef entity, NameTagComponent nameTag
             nameTag = nameTagBuilder.build();
             nameTagEntityToFloatingTextMap.put(entity, nameTag);
 
-            Location.attachChild(entity, nameTag, offset, new Quat4f(1, 0, 0, 0));
+            Location.attachChild(entity, nameTag, offset, new Quaternionf(1, 0, 0, 0));
```

Comment:
Is this a useful quaternion or a mistake? I'm tempted to say it's supposed to be the identity quaternion.